### PR TITLE
Rerun ipscan

### LINKED ISSUE #363: Add favourite sequence feature
It would be great to add a favourite sequence feature in sequence search, allowing the sequence searched to be kept and searched again when a new InterPro release is available (similar to what is provided in the new (beta) UniProt website).

### PR DESCRIPTION
**Pull request by gustavo-salazar**
Resolves #363 

A new action to re-run a job is now included in the overview of an existing job. This option is only available when a there is a version mismatch.

When the user clicks on the rerun button, the search by sequence page is open pre filling the sequence and the advanced options. The capability of pre-fill the advanced options.

### CODE REVIEW

**Review by matthiasblum — APPROVED** [review]
Looks good to me.

One question, though: wouldn't it be simpler to immediately persist results in IndexDB as soon as they are ready?

**Comment by gustavo-salazar** [comment]
> One question, though: wouldn't it be simpler to immediately persist results in IndexDB as soon as they are ready?

I think the idea is to completely separate the logic related to calling the InterProScan server, from the UI, that way the UI is only built with what's in the redux store, no matter how it got there. But in any case, if you don't like it, blame @aurel-l

File: src/store/enhancer/jobs-middleware/index.js [hunk 1]
```diff
@@ -85,6 +104,31 @@ const updateJobInDB = async (metadata, data, title) => {
...
+  let applications = null;
+  const parameters = payload?.execution?.userParameters?.entry || [];
+  parameters.forEach((parameter) => {
+    switch (parameter?.string) {
+      case 'goterms':
+        goterms = parameter?.boolean ?? null;
+        break;
+      case 'pathways':
+        pathways = parameter?.boolean ?? null;
+        break;
```
[the remark this answers is not in the document]
Comment (gustavo-salazar) [reply src/store/enhancer/jobs-middleware/index.js:121]: Yep, TBH I started the changes from there and only later remembered we had removed the `goterms` and `pathways`

File: src/styles/interpro-new.css [hunk 2]
```diff
@@ -139,6 +139,11 @@ mark {
   color: var(--colors-all);
 }
 
+.ico-neutral:hover::before {
+  opacity: 1;
+  color: var(--colors-black);
+}
```
[the remark this answers is not in the document]
Comment (gustavo-salazar) [reply src/styles/interpro-new.css:145]: Nope... That class is used in other places that aren't links/buttons. I'll add the style for the buttons in the list somewhere else

**Pull request merged**